driver/makedepend.ml

```diff
@@ -464,7 +464,7 @@ let sort_files_by_dependencies files =
 
   if !worklist <> [] then begin
     Format.fprintf Format.err_formatter
-      "@[Warning: cycle in dependencies. End of list is not sorted.@]@.";
+      "@[Error: cycle in dependencies. End of list is not sorted.@]@.";
```

Comment:
A very minor question: should this be `@{<error>Error@}` (with a `Misc.Color.setup !Clflags.color` before) to align the printing of this error with the ones printed by `Location.report_exception` (in case of parsing errors) ?

Reply:
You're probably right, but I'm not familiar with the error coloring code.  I'll merge as it, but feel free to change the error message later if you wish. ( I'm not even sure a GPR is needed for that, as it would not change functionality.)